flink/pom.xml

```diff
@@ -41,6 +41,11 @@
         <module>v1.15/flink-format</module>
         <module>v1.15/flink-runtime</module>
         <module>v1.15/arctic-flink-1.15-iceberg-bridge</module>
+        <module>v1.17/flink</module>
+        <module>v1.17/flink-runtime</module>
+        <module>v1.17/amoro-flink-1.17-iceberg-bridge</module>
```

Comment:
Can we move the classes in `iceberg-bridge` into `flink-common` rather than maintaining a separate copy for each version.

Reply:
now move to new model **flink-common-iceberg-bridge**